### PR TITLE
Created a pathway graph

### PR DESCRIPTION
**Pull request by chirag-wadhwa**

### CODE REVIEW

File: client/src/containers/Pathway/Pathway.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+            </Fragment>
+        )
+    }
+
+};
+
+export default Pathway;
+
+
+// import Graph from "react-graph-vis";
```
Comment (dusnat) [inline client/src/containers/Pathway/Pathway.js]: Come on man. Remove the comments.

**Review by dusnat — COMMENTED** [review]
Describe what you did **properly** in the PR description.

Did you just implement the pathway rendering?
Or did you make the whole page?
If you didn't make the whole page, what exactly did you make?

**Comment by chirag-wadhwa** [comment]
Created the screen to display a pathway. When a step is clicked the graph shifts to the left and the data of that step is displayed. Let me know if any changes are to be made.

**Comment by dusnat** [comment]
Also add the functionality of displaying another graph when a step that points to a graph is clicked. For now, I'm considering this feature incomplete, so I won't merge this.

**Pull request merged**